README.md

```diff
@@ -63,25 +63,25 @@ service using the D-Bus interface it provides.
 
 Beware that `d-installer` must run as root (like YaST does) to do hardware probing, partition the
 disks, install the software and so on. So you need to tell dbus about the service by copying
-`yastd/share/dbus.conf` to `/etc/dbus-1/system.d/d-installer.conf`.
+`service/config/d-installer.conf` to `/etc/dbus-1/system.d/org.opensuse.DInstaller.conf`.
 
 To run the service, type:
 
-    cd yastd
+    cd service
     bundle install
     sudo bundle exec bin/d-installer
 
 To check that `d-installer` is working, you can use a tool like
 [busctl](https://www.freedesktop.org/wiki/Software/dbus/) (or
 [D-Feet](https://wiki.gnome.org/Apps/DFeet)) if you prefer a graphical one:
 
-    busctl call org.opensuse.YaST /org/opensuse/YaST/Installer \
-      org.opensuse.YaST.Installer GetDisks
+    busctl call org.opensuse.DInstaller /org/opensuse/DInstaller/Language \
+      org.opensuse.DInstaller.Language AvailableLanguages
 
 If you want to get the properties, just type:
 
-    busctl call org.opensuse.YaST /org/opensuse/YaST/Installer \
-      org.freedesktop.DBus.Properties GetAll s org.opensuse.YaST.Installer
+    busctl call org.opensuse.DInstaller /org/opensuse/DInstaller/Language \
```

Comment:
I would say the correct paths are `/org/opensuse/DInstaller/Language1` and `org.opensuse.DInstaller.Language1`. At least, that's how the code looks like now.